### ISSUE TITLE
Custom Semiology queries Beta Database (syst_review_single_table) rather than Semio2Brain Database

### ISSUE THREAD

**Issue by thenineteen** [thenineteen/Semiology-Visualisation-Tool#182]
Related to #140:

After having updated to a new database `Semio2Brain` in resources a few weeks back, when I add a custom semiology `mappings TLd`, I still get the output as in #140 whereas if it were searching the new database I should get an error stating no such semiology found.

The output of loggings suggests `MEGA_ANALYSIS` is correctly reading the new `Semio2Brain` database (310 article etc) - is there another place where the previous version of the dataframe has been cached, maybe?

**Comment by thenineteen** [comment]
actually this was an issue that arises from setting the deafult cache behaviour to True - which was extremely confusing as it was finding semiologies that don't exist in the updated Database!

```python
def getAdvancedSettingsTab(self):
        advancedTabWidget = qt.QWidget()
        advancedTabLayout = qt.QVBoxLayout(advancedTabWidget)

        self.useCacheCheckBox = qt.QCheckBox('Use cached queries if available')
        self.useCacheCheckBox.setChecked(False)
```

I've changed this behaviour to make it False so I can continue to calibrate

**Comment by thenineteen** [comment]
fixed in 97360196857e5b86006fc51bda9d930b0b8f4bdb
